### PR TITLE
Custom validator for subscription creation

### PR DESCRIPTION
**Pull request by mpociot**

### CODE REVIEW

**Comment by aaranmcguire** [comment]
:+1:

File: resources/stubs/app/Providers/SparkServiceProvider.php [hunk 1]
```diff
@@ -152,4 +152,19 @@ protected function customizeSubscriptionPlans()
...
+    /**
+     * Customize Spark's subscription logic.
+     *
+     * @return void
+     */
+    protected function customizeSubscriptions()
+    {
+        // Spark::validateSubscriptionsWith(function (Request $request) {
+        //     return [
+        //         'name' => 'required|max:255',
```
Comment (taylorotwell) [inline resources/stubs/app/Providers/SparkServiceProvider.php]: Is this example actually valid? Wouldn't the `name` be checked during registration?

**Comment by mpociot** [comment]
Updated with the correct stub example usage.

**Comment by taylorotwell** [comment]
Please run through these features to make sure they work. You weren't passing the `$request` to your `validateSubscription` method and it was throwing an error even on a simple subscribe action so it was totally broken. I fixed it.

**Pull request merged**